Plugins/Detray/include/Acts/Plugins/Detray/DetraySurfaceGridsConverter.hpp

```diff
@@ -0,0 +1,98 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Material/MaterialSlab.hpp"
+#include "Acts/Navigation/InternalNavigation.hpp"
+#include "Acts/Utilities/BinningData.hpp"
+#include "Acts/Utilities/IAxis.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "Acts/Utilities/TypeList.hpp"
+
+#include <detray/builders/detector_builder.hpp>
+#include <detray/core/detector.hpp>
+#include <detray/definitions/geometry.hpp>
+#include <detray/io/common/geometry_reader.hpp>
+#include <detray/io/common/surface_grid_reader.hpp>
+#include <detray/io/frontend/detector_writer.hpp>
+#include <detray/io/frontend/payloads.hpp>
+#include <detray/utils/consistency_checker.hpp>
```

Comment:
Can we check, which of these includes are actually needed?
I would expect:
```suggestion

#include <detray/io/common/surface_grid_reader.hpp>
#include <detray/io/frontend/payloads.hpp>
```

Reply:
Cleaned.